src/index.js

```diff
@@ -16,11 +17,15 @@ if (process.env.NODE_ENV !== "production") {
 
 const mapConfig = parseUriHash(defaultMapConfig);
 
+new InfoWindow();
```

Comment:
@jimboweb these class creations should be moved into the `i18nInit().then()` anonymous function with the others. Basically we need to wait for the `i18n` library to finish loading before we can run translations as it loads async.

Reply:
This was an artifact of the merge process. Moved them to the init.then() function